pkg/sign/keys_test.go

```diff
@@ -26,6 +27,7 @@ func Test_EphemeralKeypair(t *testing.T) {
 		Hint: []byte("asdf"),
 	}
 
+	ctx := context.TODO()
```

Comment:
For tests, `context.Background()` is more appropriate.

```suggestion
	ctx := context.Background()
```

Reply:
@codysoyland I opted for `TODO` cause that's what I saw used in other tests:

https://github.com/sigstore/sigstore-go/blob/3280a6017c880f6a4288ee43759b828977a1a810/pkg/sign/transparency_test.go#L106

https://github.com/sigstore/sigstore-go/blob/797ca59e9032c12c7e4a39092eb91c9d7c1b3b63/pkg/sign/timestamping_test.go#L82

Would you still like me to change it?